portal/app/[locale]/_components/navbar/_components/help.tsx

```diff
@@ -135,10 +136,14 @@ const ItemWithSubmenu = function ({
 const LanguageMenu = function ({ active }: LanguageProps) {
   const t = useTranslations('navbar.help')
   const pathname = usePathnameWithoutLocale()
+  const searchParams = useSearchParams()
   const router = useRouter()
 
   const onClick = function (locale: Locale) {
-    router.push(pathname, { locale })
+    const query = searchParams.toString()
+    const fullPath = query ? `${pathname}?${query}` : pathname
+
+    router.push(fullPath, { locale })
```

Comment:
I know this was already like that, but I just noticed it was incorrect

```suggestion
    const query = searchParams.toString()
    const fullPath = query ? `${pathname}?${query}` : pathname

    router.push(fullPath)
```

`locale` is a path Parameter, not a query string (which is the second argument of `router.push`). It should be removed